app/models/manageiq/providers/kubernetes/container_manager/metrics_capture.rb

```diff
@@ -37,8 +37,19 @@ class TargetValidationError < RuntimeError; end
       }
     }
 
+    def fetch_target_authentication
+      ems = target.ext_management_system
+      return unless ems
+
+      auth = ems.authentication_type(:metrics_collection)
+      ems.authentications << Authentication.new(:authtype => :metrics_collection) unless auth
```

Comment:
@moolitayer 
> [ is this ] ... what @simon3z and @blomquisg agreed ... ?

No, It is the best solution I found so far, It does need to be discussed, and may be a better solution will show itself ...